api/challenge-response/schemas/components.yaml

```diff
@@ -46,9 +46,10 @@ components:
           description: >
             the submitted attestation Evidence
         result:
-          $ref: '#/components/schemas/AttestationResult'
+          type: string
           description: >
-            the computed Attestation Result
+            the computed Attestation Result wrapped as payload in an JWS.
+            The wrapped Attestation Result is of type '#/components/schemas/AttestationResult'
```

Comment:
```suggestion
            the computed Attestation Result as an EAR (https://datatracker.ietf.org/doc/draft-fv-rats-ear/).
```

Reply:
@thomas-fossati I would `prefer` a statement with slightly more clarity (see below):

the computed EAT Attestation Result (EAR) wrapped as payload in an JWS.
(https://datatracker.ietf.org/doc/draft-fv-rats-ear/)

Reply:
> the computed EAT Attestation Result (EAR) wrapped as payload in an JWS. (https://datatracker.ietf.org/doc/draft-fv-rats-ear/)

Unfortunately, the proposed text is not clearer, on the contrary, it's an incorrect statement: the EAR is not _"[w]rapped as payload in a JWS"_, *it is* a JWT with a well-defined claims set.

Reply:
Yes it is a JWT. The proposed text indicates that it is a EAT Attestation Result, which is not correct.

Then I would suggest we state that the:

computed EAT Attestation Result Claims conveyed in a JWT:
(https://datatracker.ietf.org/doc/draft-fv-rats-ear/)

Reply:
> Yes it is a JWT. The proposed text indicates that it is a EAT Attestation Result, which is not correct.

It *is* correct.  What we could add is it's serialised as JWT (rather than CWT).

Reply:
The original text gave an impression that the string is a conversion of EAR as a string. If you see EAR repo it talks about EAT Attestation Result (defining a definite structure of EAT Attestation Result). Hence the statement was incorrect.

What your new text is a slight improvement. Ideally you would start from top, i.e. what the top level structure is: which is JWT which has a payload in which EAR Claims are encoded.

Reply:
> > The original text gave an impression that the string is a conversion of EAR as a string. If you see EAR repo it talks about EAT Attestation Result (defining a definite structure of EAT Attestation Result). Hence the statement was incorrect.
> 
> No, again, the suggested text, as well as the modified one, is correct. In particular, the serialised EAR _is_ a text string.
> 
> > What your new text is a slight improvement. Ideally you would start from top, i.e. what the top level structure is: which is JWT which has a payload in which EAR Claims are encoded.
> 
> I think you are misunderstanding what an EAR is. EAR is an info model (wrapping the AR4SI info model), which happens to have two possible serialisations: JWT and CWT. JWT is not _"[t]he top-level structure"_, it's just a wire format (and security envelope).

The receiver of the `string` for  `ChallengeResponse Session` is going to receive a `JWT` hence it has to  Verify `JWT` to mean something out of it, isn't it. It is very important we say that explicitly in the interface.
Yes we have encoded EAR inside a `JWT` for sure.

Also EAR has a concrete structure, so it has both Models, `Info` and `Data`? Why you say only `info` model?

Reply:
I think the current suggestion correctly captures the intended semantics.